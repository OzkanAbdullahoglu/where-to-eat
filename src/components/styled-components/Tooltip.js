/* eslint-disable linebreak-style */
import styled from 'styled-components';
import { PanelToggleButton } from './PanelToggleButton';

export const Tooltip = styled.span`
         display: none;
         ${PanelToggleButton}:hover & {
           display: block;
           z-index: 100;
           padding: 2px 8px;
           border-radius: 6px;
           white-space: nowrap;
           background-color: #2b6aff;
           color: #eff0f3;
           font-family: Arial, Helvetica, sans-serif;
           font-size: 1rem;
           font-weight: 400;
           line-height: 1.25rem;
           box-shadow: 0 4px 10px rgba(170, 170, 170, 0.75);
           position: absolute;
           left: 100%;
           top: 50%;
           -webkit-transform: translateY(-50%);
           transform: translateY(-50%);
           margin-left: 8px;
           line-height: 22px;
           white-space: nowrap;
           transition-duration: 0.3s;
           transition-timing-function: linear;
         }
       `;
