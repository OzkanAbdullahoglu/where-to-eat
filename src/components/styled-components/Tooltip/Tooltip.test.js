/* eslint-disable linebreak-style */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import Tooltip from './Tooltip';

describe('Search Box Container Render', () => {
  it('renders search box container', () => {
    const toolTipWrapper = shallow(<Tooltip />);
    expect(toJson(toolTipWrapper)).toMatchSnapshot();
  });
});
