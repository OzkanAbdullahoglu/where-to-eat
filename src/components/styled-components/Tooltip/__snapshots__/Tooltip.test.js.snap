// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Box Container Render renders search box container 1`] = `
.c1 {
  display: none;
}

.c1:hover .c0 {
  display: block;
  z-index: 100;
  padding: 2px 8px;
  border-radius: 6px;
  white-space: nowrap;
  background-color: #2b6aff;
  color: #eff0f3;
  font-family: Arial,Helvetica,sans-serif;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.25rem;
  box-shadow: 0 4px 10px rgba(170,170,170,0.75);
  position: absolute;
  left: 100%;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  margin-left: 8px;
  line-height: 22px;
  white-space: nowrap;
  -webkit-transition-duration: 0.3s;
  transition-duration: 0.3s;
  -webkit-transition-timing-function: linear;
  transition-timing-function: linear;
}

<span
  className="c0 c1"
/>
`;
